test/fixtures/cli/Gemfile

```diff
@@ -0,0 +1,4 @@
+source "https://rubygems.org"
```

Comment:
Is this file needed for anything?

Reply:
Is it so we can run `bundle exec maze-runner` in the local directory?